joi-validations/topics.validations.js

```diff
@@ -0,0 +1,11 @@
+const {string} = require('./general.validations');
+
+const TopicValidation = {
+  latestPost: {
+    query: {
+      name: string.required()
+    }
+  }
+};
+
+module.exports = {TopicValidation};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The validation logic for the `latestPost` endpoint seems fine. It correctly requires a `name` field in the query parameters, which is necessary to fetch the latest post topics based on a given topic name. However, it would be better if you also validate the length of the string to prevent any potential SQL injection attacks or errors due to excessively long strings.
```diff
      name: string.required()
+       name: string.required().max(255)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->